js/scripts.js

```diff
@@ -0,0 +1,127 @@
+var computerResultElem = document.getElementById('js-computerResult');
+
+var gameState = 'notStarted'; 
+var player = {name: '', score: 0};
+var computer = {score: 0};
+ 
+newGameBtn.addEventListener('click', newGame);
+
+pickRock.addEventListener('click', function() { playerPick('rock') });
+pickPaper.addEventListener('click', function() { playerPick('paper') });
+pickScissors.addEventListener('click', function() { playerPick('scissors') });
+
+
+
+
+function setGameElements() {
+  switch(gameState) {
+    case 'started':
+        newGameElem.style.display = 'none';
+        pickElem.style.display = 'block';
+        resultsElem.style.display = 'block';
+      break;
+    case 'ended':
+        newGameBtn.innerText = 'Jeszcze raz?';
+    case 'notStarted':
+    default:
+        newGameElem.style.display = 'block';
+        pickElem.style.display = 'none';
+        resultsElem.style.display = 'none';
+  }
+}
+
+
+function newGame() {
+  player.name = prompt('Graczu, wpisz swoje imię', 'imię gracza');
+  if (player.name) {
+    player.score = computer.score = 0;
+    gameState = 'started';
+    setGameElements();
+
+    playerNameElem.innerHTML = player.name;
+    setGamePoints(); 
+  }
+
+}
+function playerPick(playerPick) {
+    console.log(playerPick);
+    var computerPick = getComputerPick();
+}
+function getComputerPick() {
+    var possiblePicks = ['rock', 'paper', 'scissors'];
+    return possiblePicks[Math.floor(Math.random() * 3)];
+}
+
+
+function checkRoundWinner(playerPick, computerPick) {
+  playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+  var winnerIs = 'player';
+
+    if (playerPick === computerPick) {
+        winnerIs = 'noone'; 
+		playerResultElem.innerHTML = "Remis!";
+		computerResultElem.innerHTML = "Remis!";
+    } else if (
+        (computerPick === 'rock' &&  playerPick === 'scissors') ||
+        (computerPick === 'scissors' &&  playerPick === 'paper') ||
+        (computerPick === 'paper' &&  playerPick === 'rock') ) {
+        
+        winnerIs = 'computer';
+    }
+
+
+
+    if (winnerIs === 'player') {
+        playerResultElem.innerHTML = "Punkt dla Ciebie!";
+        player.score++;
+        playerPointsElem.innerHTML = player.score;
+    } else if (winnerIs === 'computer') {
+        computerResultElem.innerHTML = "Punkt dla Ciebie!";
+        computer.score++;
+        computerPointsElem.innerHTML = computer.score;
+    }
+}
+/*function checkEndOfGame(score) {
+    <<-- (score) (player, score), (playerPointsElem, computerPointsElem), (player.score, computer.score)-->>
+    playerPointsElem.innerHTML = player.score;
+    computerPointsElem.innerHTML = computer.score;
+    winnerIs = 'player';*/
+	if (player.score === 3) {
+        alert('Wygrałeś!');
+        gameState = 'ended';
+        setGameElements()
+	} else if (computer.score === 3) {
+        alert('Wygrał przeciwnik!');
+        gameState = 'ended';
+        setGameElements()
+	}
+	    
+}
+
```

Comment:
ale namotał :P

Twoja funkcja

```
function checkEndOfGame() {
	if (player.score === 3) {
        alert('Wygrałeś!');
        gameState = 'ended';
        setGameElements()
	} else if (computer.score === 3) {
        alert('Wygrał przeciwnik!');
        gameState = 'ended';
        setGameElements()
	}
}
```

i wywołujesz ją na końcu `checkRoundWinner` i tyle :)

Jak Ci się uda już to możesz pokombinować jak w `checkEndOfGame()` pozbyć się DRY.